gulpfile.js

```diff
@@ -176,10 +210,12 @@ var dmg_cmd = template('./osx/create-dmg/create-dmg --icon "<%= name %>" 311 50
 
 // Only runs on OSX (requires XCode properly configured)
 gulp.task('sign:osx', ['build'], shell.task([
+  /* */
```

Comment:
`gulp build:osx` will fail if you don't have the code signing credentials in your keychain (I don't) so this acts as a nice comment toggle. Would be happy to remove if we get some proper documentation on signing this code and setup on this.